src/work-allocation-2/containers/task-list/task-list.component.html

```diff
@@ -1,101 +1,98 @@
-<div class="task-list-container">
-  <div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
-    <span class="text-16" id="search-result-summary__text">Displaying
-        <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
-        -
-        <span class="govuk-!-font-weight-bold">{{ getLastResult() }}</span>
-        out of
-        <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }}</span>
-        tasks.</span>
-  </div>
+<div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
+  <span class="text-16" id="search-result-summary__text">Displaying
+      <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
```

Comment:
_Nitpick:_
This block is indented too far.

Reply:
I've formatted the whole file now (!)